releng_changelog.md

```diff
@@ -6,14 +6,19 @@ This page best serves the people that have previously been into the releaseduty
 
 _**As releaseduty squirrels are the ones with the best context when it comes to releases, they are the ones to edit this page and amend it accordingly. Keep in mind that changes should be in compliance with the other pieces of documentation.**_
 
+## During 82.0 >= 2020-09-21
+### Changed
+- partner attribution configs landed
+- balrog v2 submission
```

Comment:
v2 submission will ride with 83 -- should it be in a different section?

Reply:
I never really understood the version thing, and some changelog items might not be for gecko, so the version might be completely wrong. I generally just follow the date. We may want to revisit how we lay out the changelog.

Reply:
:shipit: